README.md

```diff
@@ -2,9 +2,32 @@
 
 Operator to manage deadmanssnitch configs for Openshift Dedicated
 
+- [deadmanssnitch-operator](#deadmanssnitch-operator)
+  - [Overview](#overview)
+  - [Metrics](#metrics)
+  - [Alerts](#alerts)
+  - [Usage](#usage)
+
+## Overview
+
+The operator runs on hive. It has a single controller. It:
+- Requires a master Secret to talk to the Dead Man's Snitch API.
+  This secret is expected to be named `deadmanssnitch-api-key` and live in the `deadmanssnitch-operator` namespace.
```

Comment:
Is it hard coded to a namespace or is it expected to be co-located with the operator deployment?

Reply:
[Hard](https://github.com/openshift/deadmanssnitch-operator/blob/5a5465c0ba91df454218991477b61e6aa4aa4b79/pkg/controller/deadmanssnitch/deadmanssnitch_controller.go#L67-L69) [coded](https://github.com/openshift/deadmanssnitch-operator/blob/5a5465c0ba91df454218991477b61e6aa4aa4b79/pkg/controller/deadmanssnitch/deadmanssnitch_controller.go#L214-L215) to [constants](https://github.com/openshift/deadmanssnitch-operator/blob/5a5465c0ba91df454218991477b61e6aa4aa4b79/pkg/controller/deadmanssnitch/deadmanssnitch_controller.go#L33-L36)